etest/build.py

```diff
@@ -1,32 +1,105 @@
 """Build command for etest Docker images."""
-import os
-import sys
+from enum import Enum
 from pathlib import Path
-from typing import List
 
 import click
 
+from etest import docker, qemu
+from etest.profile import Profile
+
+
+class __commands(Enum):
+    glibc = """
```

Comment:
If you use textwrap.dedent you can make the indentation look much nicer for these.